app/backend/src/tests/test_requests.py

```diff
@@ -51,15 +51,6 @@ def test_create_request(db):
         assert e.value.code() == grpc.StatusCode.INVALID_ARGUMENT
         assert e.value.details() == errors.DATE_FROM_AFTER_TO
 
-        with pytest.raises(grpc.RpcError) as e:
-            api.CreateHostRequest(
-                requests_pb2.CreateHostRequestReq(
-                    to_user_id=user2.id, from_date=today_plus_2, to_date=today_minus_2, text="Test request"
-                )
-            )
-        assert e.value.code() == grpc.StatusCode.INVALID_ARGUMENT
-        assert e.value.details() == errors.DATE_TO_BEFORE_TODAY
-
```

Comment:
You can leave the test as it is, just don't be so picky about the error text. (remove the e.value.details() check)

That way, we can be sure that the logic is still the same despite the removed code.

Reply:
Makes sense